### DISCUSSION TITLE
Lack of moderation on matrix.org

### DISCUSSION THREAD

**Discussion by konomikitten** [element-hq/element-meta#2485] [General]
I and many other users are receiving a constant stream of abusive invite spam originating from the matrix.org instance. I've sent multiple emails to abuse@matrix.org but these abusive room invites continue. Why does there seem to be a incredible lack of moderation on the part of the matrix organization on their own server?

**Comment by konomikitten** [comment]
I've forwarded 9 screenshots of abusive room invites to you guys, really would like you to do something about this targeted harassment.

**Reply by awtj8o81ryywg793** [reply]
Element isn't the Matrix.org Foundation, and as such, doesn't moderate the matrix.org homeserver. Given that fact, I would recommend closing this discussion.

Considering that this discussion started over a year ago, a good amount has changed regarding trust and safety. You can read more about it here: https://matrix.org/category/trust-safety/
